templates/andrzejki.html

```diff
@@ -0,0 +1,61 @@
+<html>
+    <head>
+        <title>Andrzejki</title>
+        <style>
+            body { background-color: black; }
+            .images {
+                display: grid;
+                grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+                justify-items: center;
+                align-items: center;
+                height: 100vh;
+                width: 100%;
+                max-height: 100%;
+                padding: 0;
+            }
+            img {
+                width: calc(100% - 1em);
+                height: auto;
+                object-fit: contain;
+                image-rendering: pixelated;
+            }
+        </style>
+        <script>
+            const IMAGES_COUNT = 5;
+            const URL = "/image.png";
+            const TIME = 5000;
+            let timeout = null;
+
+            function start() {
+                clear();
+                load_images(0);
+            }
+
+            function clear() {
+                if (timeout !== null) {
+                    clearTimeout(timeout);
+                    timeout = null;
+                }
+                const images = document.querySelector(".images");
+                images.innerHTML = "";
+            }
+
+            function load_images(i) {
+                timeout = setTimeout(() => load_images(i + 1), TIME);
+                if (i == 0 || i > IMAGES_COUNT) {
+                    return;
+                }
+                const timestamp = Date.now();
+                const images = document.querySelector(".images");
+                const image = document.createElement("img");
+                image.alt = "pixelart";
+                image.src = URL + "?" + timestamp + "--" + i;
```

Comment:
I wonder why do you need timestamp. Is index not enough to make this resource uniq?